website/templates/include/comment_pane_template.mako

```diff
@@ -24,14 +24,14 @@
                 <form class="form">
                     <div class="form-group">
                         <span>
-                            <textarea class="form-control" placeholder="Add a comment" data-bind="value: replyContent, valueUpdate: 'input', attr: {maxlength: $root.MAXLENGTH}"></textarea>
+                            <div class="form-control atwho-input" placeholder="Add a comment" data-bind="editableHTML: replyContent, attr: {maxlength: $root.MAXLENGTH}" contenteditable="true"></div>
```

Comment:
That may depend to some extent on the browser. I'm seeing `cmd-b` do one thing in Chrome, another in Firefox. Maybe Firefox on Mac OS has a different keyboard shortcut? I'm also seeing bold + italics do something unexpected on Chrome. To avoid a cross-browser morass, perhaps there's a way to strip un-whitelisted HTML on the client-side?